contracts/DifficultyOracle.sol

```diff
@@ -10,17 +10,15 @@ contract DifficultyOracle {
     uint public endTime;
     /// Difficulty level at which oracle would report a full value in the first slot if the actual difficulty was less than or equal to this target, and a full value in the second slot if the actual difficulty exceeded the target.
     uint public difficultyTarget;
-    /// Question ID oracle will use during report to the prediction market system. (AUDIT: this has no accessor unlike the other variables?)
-    bytes32 questionId;
+    /// Question ID oracle will use during report to the prediction market system. 
+    bytes32 public questionId;
 
     /// @dev Emitted upon the successful reporting of whether the difficulty has exceeded the difficulty target to the prediction market system.
-    /// @param startTime Beginning of time window in which valid reports may be generated. (AUDIT: duplicate of storage var)
-    /// @param endTime End of time window in which valid reports may be generated. (AUDIT: duplicate of storage var)
+    /// @param _startTime Beginning of time window in which valid reports may be generated. 
+    /// @param _endTime End of time window in which valid reports may be generated.
```

Comment:
My apologies for being vague here. What I had meant to express was not that the name wasn't good, because the names are fine, but rather that `startTime` and `endTime` is immutable and already available via accessors. In that sense, they are similar to the `difficultyTarget`, but `difficultyTarget` isn't echoed here as an event parameter, so I figured either add `difficultyTarget` to the event for consistency or remove `startTime` and `endTime`, reporting only the `currentTime` and the witnessed `difficulty` value (this would be cheaper). This applies of course to the other events similar to this one.